### PR TITLE
Add control and art pane components

### PR DESCRIPTION
**Pull request by leggomyfroggo**
Super basic functionality. List pane can be expanded/collapsed and that's it.

### CODE REVIEW

File: src/components/ControlPane.tsx [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+export class ControlPane extends React.Component<IControlPaneProps, IControlPaneState> {
+  public state: IControlPaneState = {isExpanded: false};
+
+  public render(): React.ReactNode {
+    return (
+      <div className="control-pane">
+        {this.renderListPane()}
+        <div className="control-bar">
+          <div onClick={this.onListButtonClicked}>EXPAND</div>
```
Comment (dxprog) [inline src/components/ControlPane.tsx]: This should be a `<button>`

File: src/components/ControlPane.tsx [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+  private onListButtonClicked = (): void => {
+    this.setState({isExpanded: !this.state.isExpanded});
+  }
+
+  private renderListPane(): React.ReactNode {
+    if (!this.state.isExpanded) {
+      return null;
+    }
+    return <div className="list-pane" />;
```
Comment (dxprog) [inline src/components/ControlPane.tsx]: since this is simple logic: `return !this.state.isExpanded ? null : <div className="list-pane" />;`

**Review by dxprog — APPROVED** [review]
Couple thoughts, no blockers. Merge away.

**Comment by leggomyfroggo** [comment]
Updated with suggestions.

**Pull request merged**